src/vinylcontrol/vinylcontrol.cpp

```diff
@@ -40,6 +40,7 @@ VinylControl::VinylControl(UserSettingsPointer pConfig, const QString& group)
     signalenabled = new ControlProxy(
             group, "vinylcontrol_signal_enabled", this);
     reverseButton = new ControlProxy(group, "reverse", this);
+    passthroughEnabled = new ControlProxy(group, "passthrough", this);
```

Comment:
This can become: 
```suggestion
    passthroughEnabled = make_parented<ControlProxy>(group, "passthrough", this);
```

or even better: 
PollingControlProxy m_passthroughEnabled;

because it has no connection 

(Sorry, you have touched old code that is implements our recent style guide we like to introduce for changes.

Reply:
@daschuer With `PollingControlProxy`, this will be instantiated as the following, right?
```
passthroughEnabled(group, QStringLiteral("passthrough"));
```

Reply:
Yeah, let's update to `parented_ptr`, switch to `PollingControlProxy` and add `m_p`/`m_` prefix when all functional changes have been implemented and this PR got approved.
Wdyt?

Reply:
Hi @ronso0!
Got it, sorry, I completely missed that part. I will do it very soon.

Reply:
No hurry..

Reply:
@ywwg I made the change. The PR is ready.